cpp/example_code/new_s3/CMakeLists.txt

```diff
@@ -0,0 +1,84 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.2)
+project(s3-examples)
+
+set (CMAKE_CXX_STANDARD 11)
+
+# Enable CTest
+include(CTest)
+
+# Build as dynamic libraries by default
+if(NOT BUILD_SHARED_LIBS)
+    set(BUILD_SHARED_LIBS ON)
+endif()
+
+# Link to shared libraries.
+if(MSVC AND BUILD_SHARED_LIBS)
+    add_definitions(-DUSE_IMPORT_EXPORT)
+endif()
+
+# Locate the aws sdk for c++ package.
+find_package(AWSSDK REQUIRED COMPONENTS s3)
+
+# Header files
+file(GLOB AWSDOC_S3_HEADERS
+    "include/awsdoc/s3/*.h"
+)
+
+# Source files
+file(GLOB AWSDOC_S3_SOURCE
+    "source/*.cpp"
```

Comment:
Could this be changed to simply "*.cpp"? This would make it a bit easier for customers to find the code examples, without forcing them to click into a "source" folder.